redisinsight/ui/src/components/message-bar/MessageBar.tsx

```diff
@@ -16,25 +18,45 @@ const MessageBar = ({ children, opened }: Props) => {
     setIsOpen(opened)
   }, [opened])
 
-  return isOpen ? (
-    <div className={styles.inner}>
-      <div className={styles.containerWrapper}>
-        <Row centered className={styles.container} gap="l">
-          <FlexItem grow className={styles.text}>
-            {children}
-          </FlexItem>
-          <FlexItem className={styles.cross}>
-            <IconButton
-              icon={CancelSlimIcon}
-              aria-label="Close"
-              onClick={() => setIsOpen(false)}
-              data-testid="close-button"
-            />
-          </FlexItem>
-        </Row>
-      </div>
-    </div>
-  ) : null
+  if (!isOpen) {
+    return null
+  }
+  return (
+    <ContainerWrapper centered>
+      <Container grow={false} centered gap="l">
+        <FlexItem grow className={styles.text}>
+          {children}
+        </FlexItem>
+        <FlexItem className={styles.cross}>
+          <IconButton
+            icon={CancelSlimIcon}
+            aria-label="Close"
+            onClick={() => setIsOpen(false)}
+            data-testid="close-button"
+          />
+        </FlexItem>
+      </Container>
+    </ContainerWrapper>
+  )
 }
 
+const Container = styled(Row)`
```

Comment:
nit: Minor, but I guess you forgot to move these styles into the usual `MessageBar.styles.ts` file, once you created them.